servers/services/src/main/java/org/projectnessie/services/rest/TreeResource.java

```diff
@@ -405,11 +433,15 @@ protected Hash doOps(
       List<org.projectnessie.versioned.Operation<Contents>> operations)
       throws NessieConflictException, NessieNotFoundException {
     try {
+      if (commitMeta.getCommitter() != null) {
+        throw new NessieConflictException(
+            "Cannot set the committer on the client side. It is set by the server.");
+      }
       return getStore()
           .commit(
               BranchName.of(Optional.ofNullable(branch).orElse(getConfig().getDefaultBranch())),
               Optional.ofNullable(hash).map(Hash::of),
-              meta(getPrincipal(), commitMeta),
+              meta(commitMeta, Collections.EMPTY_MAP),
```

Comment:
nit: `Collections.emptyMap()`

Reply:
Isn't it same?

```
 public static final <K, V> Map<K, V> emptyMap() {
        return EMPTY_MAP;
    }
```

Reply:
It is not. Generics.

Reply:
Removing the code related to properties, this param will not be needed